ui/src/partials/toolbar.hbs

```diff
@@ -6,10 +6,13 @@
   {{/with}}
   {{> breadcrumbs}}
   {{> page-versions}}
+  {{#unless (eq page.attributes.theme "docs") }}
```

Comment:
If we want to do that I think we should use an explicit attribute.

Please note that if the documentation was fetched from this repository by Antora, it would just work out-of-the-box because the repository `neo-technology/neo4j-manual-modeling-antora` is private.
In other words, `page.origin.private` will be true:

> If a content source requires authentication (either because credentials were defined in the URL or credentials were requested from the credentials manager), the page.origin.private property will be set in the UI model.

https://docs.antora.org/antora/2.0/whats-new/#more-improvements-and-changes

Reply:
The documentation is not fetched from any repo by Antora at the moment. The asciidoc requires pre-processing before we can run Antora and we don't commit the processed files. Instead our build clone the repo to get the playbook, takes the source from artifacts of other builds in the dependency chain, and uses [the git worktree](https://docs.antora.org/antora/2.3/playbook/content-source-url/#git-worktree).

How would we go about setting an explicit attribute?

Reply:
> How would we go about setting an explicit attribute?

In my opinion, the theme attribute is not tightly related since a "public" documentation component could also use the theme "docs" (because it looks good).

I think we should add an attribute named `page-private` or `page-origin-private` in the playbook (`antora-playbook-default.json`) and update the condition `{{else if (and page.editUrl (or env.FORCE_SHOW_EDIT_PAGE_LINK (not page.origin.private)))}}` to check if the page is private (i.e. `page.origin.private` is true or `page.attributes.origin-private` is true).

Reply:
I don't have a particular preference but I'm happy to go with whatever seems more 'correct'. Using an attribute does have the advantage of allowing us to override the value, but that's only a future possibility as long as we are not fetching content from a repo.